website/docs/learn/1-setting-up.md

```diff
@@ -4,23 +4,24 @@ id: setting-up
 description: This tutorial is the required pre-work for dbt Learn.
 ---
 
+
 This tutorial is the required pre-work for dbt Learn. Before beginning this tutorial, please ensure that you been received your Snowflake credentials and dbt Cloud invitation by email from a Learn instructor.
 
-If you are **new to dbt**, we recommend you take some time to read through the FAQs, and come armed with any questions you have along the way. If you get completely stuck in this tutorial, please [email us](mailto:learn@getdbt.com) and we'll help unstick you.
+If you are **new to dbt**, we recommend you take some time to read through the FAQs, and come armed with any questions you have along the way. If you get completely stuck in this tutorial, please ask questions in the dedicated Slack channel, as detailed in the welcome email.
 
```

Comment:
Yes! please send questions to the slack - the sooner the better!

...where does learn@getdbt.com go anyways?

Reply:
Does it not... go to you?

Reply:
I just sent an email to learn@getdbt.com - nothing coming through to kcoapman@getdbt.com